cmd/tests/snapshots/output/missing-symbol-information/dump.lsif

```diff
@@ -0,0 +1,37 @@
+{"id":1,"version":"0.4.3","projectRoot":"file:/root","positionEncoding":"utf-8","toolInfo":{"name":"reprolang","version":"1.0.0"},"type":"vertex","label":"metaData"}
+{"id":2,"type":"vertex","label":"definitionResult"}
+{"id":3,"type":"vertex","label":"resultSet"}
+{"id":4,"type":"vertex","label":"referenceResult"}
+{"id":5,"type":"vertex","label":"hoverResult","result":{"contents":{"kind":"markdown"}}}
+{"id":6,"type":"edge","label":"textDocument/definition","inV":2,"outV":3}
+{"id":7,"type":"edge","label":"textDocument/references","inV":4,"outV":3}
+{"id":8,"type":"edge","label":"textDocument/hover","inV":5,"outV":3}
+{"id":9,"type":"vertex","label":"moniker","identifier":"reprolang repro_manager missing-symbol-information 1.0.0 globals.repro/NoSymbolInformation#","kind":"export","scheme":"reprolang"}
```

Comment:
The bugfix was that this moniker is now `"kind":"export"` instead of `"kind":"import"`